tools/ml-dataprep/bigquery_test.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for the bigquery module."""
+
+
+import bigquery
+
+
+BQ_CLIENT = bigquery.BqClient()
+
+
+def test_build_table_uri():
```

Comment:
I would recommend to use standard unittest package

Reply:
Done in 34720691e7411adb8ac2da902d64afa0d825e733